test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -533,6 +539,55 @@ private String summarizeStartInstructions(
     }
   }
 
+  private String summarizeProcessInstanceModification(final Record<?> record) {
+    final var value = (ProcessInstanceModificationRecordValue) record.getValue();
+    return new StringBuilder()
+        .append(summarizeActivateInstructions(value.getActivateInstructions()))
+        .append(summarizeTerminateInstructions(value.getTerminateInstructions()))
+        .toString();
+  }
+
+  private String summarizeActivateInstructions(
+      final List<ProcessInstanceModificationActivateInstructionValue> activateInstructions) {
+    if (activateInstructions.isEmpty()) {
+      return "";
+    }
+    return activateInstructions.stream()
+        .map(
+            a ->
+                "activate "
+                    + formatId(a.getElementId())
+                    + " "
+                    + summarizeVariableInstructions(a.getVariableInstructions()))
+        .collect(Collectors.joining("> <", "<", "> "));
+  }
+
+  private String summarizeVariableInstructions(
+      final List<ProcessInstanceModificationVariableInstructionValue> variableInstructions) {
+    if (variableInstructions.isEmpty()) {
+      return "no vars";
+    }
+    final var builder = new StringBuilder().append("with vars ");
+    variableInstructions.forEach(
+        v -> {
+          if (v.getElementId() != null) {
+            builder.append("@").append(formatId(v.getElementId()));
+          }
+          builder.append(v.getVariables());
+        });
+    return builder.toString();
+  }
+
+  private String summarizeTerminateInstructions(
+      final List<ProcessInstanceModificationTerminateInstructionValue> terminateInstructions) {
+    if (terminateInstructions.isEmpty()) {
+      return "";
+    }
+    return terminateInstructions.stream()
+        .map(t -> "terminate " + formatKey(t.getElementInstanceKey()))
```

Comment:
:wrench: We could align the formatting of the element instance key with other places.

In other places, we use `shortenKey()` instead of `formatKey()` and the key is often wrapped into brackets `[key]`. See [summarizeElementInformation()](https://github.com/camunda/zeebe/blob/cace4ceb4c287c1f33b66079e50214cb3172ef9b/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java#L339).

Reply:
Oh interesting, I wasn't aware of this. I guess I used the wrong method. But, that's a good point 👍